server/plugins/connect-plugin-api/src/main/java/com/oceanbase/odc/plugin/connect/model/UserRole.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.plugin.connect.model;
+
+/**
+ * @author jingtian
+ * @date 2023/11/13
+ * @since
+ */
+public enum UserRole {
```

Comment:
this should only for oracle, may: `server/plugins/connect-plugin-api/src/main/java/com/oceanbase/odc/plugin/connect/model/oracle/UserRole.java`